.flake8

```diff
@@ -1,7 +1,7 @@
 [flake8]
 ignore = W,
     E24,E121,E123,E125,E126,E221,E226,E266,E704,
-    E265,E722,E501,E731,E306,E401,E302,E222
+    E265,E722,E501,E731,E306,E401,E302,E222,E303
```

Comment:
This is the one that complains about incorrect number of blank lines between things. It dost protest too much...